src/lucky/routable.cr

```diff
@@ -145,16 +145,17 @@ module Lucky::Routable
     # no-op by default
   end
 
-  NORMALIZED_ROUTES = [] of Nil # types are not resolved in macros
+  private NORMALIZED_ROUTES = {} of _ => _
```

Comment:
I think we can keep this `Nil => Nil`. That's pretty common in other areas. I don't think we use the `_` often outside of maybe a Proc. Plus with the recent commit on Crystal, I'm not sure if the Underscore stuff will change or not.

Reply:
FWIW: Using underscore as type argument is IMO ideal because it clearly designates this constant as used at compile-time only.

Reply:
If @straight-shoota is cool with it, then that's good enough for me!